src/lang/korean.txt

```diff
@@ -383,7 +383,7 @@ STR_SCENEDIT_FILE_MENU_QUIT                                     :게임 종료
 
 ############ range for settings menu starts
 STR_SETTINGS_MENU_GAME_OPTIONS                                  :게임 기본 설정
-STR_SETTINGS_MENU_CONFIG_SETTINGS_TREE                          :설정
+STR_SETTINGS_MENU_CONFIG_SETTINGS_TREE                          :고급 설정
```

Comment:
This is intended translation. The original string is just "Settings" which of its older string was "Advanced settings"